puzzles/solutions/2022/d02/p1.py

```diff
@@ -0,0 +1,42 @@
+import sys
+
+import shapes
+
+
+def get_moves(input_text: str) -> list[tuple[str, str]]:
```

Comment:
Return `tuple[tuple[str, str]]` if you are not going to mutate the list.

Reply:
Fixed in 88065621.